public/index.html

```diff
@@ -23,7 +25,10 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
-    <title>Music School</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
```

Comment:
This is not necessary anymore, we can pass to table through `icons` props.

[Example 1/2](https://github.com/ufv-crp/Music/blob/users/src/pages/users/users.js#L57)
[Example 2/2](https://github.com/ufv-crp/Music/blob/users/src/pages/users/tableData.js#L21)

Reply:
@iguit0 , done. I have create a component on the component' folder, that is simplified, because the Material Table doesn't require the icons to be referenced, as the examples that you sent... [here](https://github.com/ufv-crp/Music/pull/64/commits/32bc72f89ce88300524bce0650f7654d8b8e3f30#diff-8b371a6fa195010ff94c7214882523b7) how it loks like now